src/c++/perf_analyzer/test_concurrency_manager.cc

```diff
@@ -30,52 +30,13 @@
 #include "doctest.h"
 #include "mock_client_backend.h"
 #include "mock_data_loader.h"
+#include "mock_infer_data_manager.h"
 #include "mock_model_parser.h"
 #include "test_load_manager_base.h"
 #include "test_utils.h"
 
 namespace triton { namespace perfanalyzer {
 
-class ConcurrencyWorkerMockedInferInput : public ConcurrencyWorker {
```

Comment:
Basically this was some ugliness to get around the fact that this work wasnt in place when the mock infer input was necessary. Now that the infer_data_manager handles creating the infer_input, this hierarchy isnt necessary anymore since it can be mocked from the infer_data_manager class